views/layout.pug

```diff
@@ -27,7 +27,7 @@ html
         function update_stats(){
           $.ajax({url: '/ext/summary', success: function(json){
             $("#supply").text(parseInt(parseFloat(json.data[0].supply).toFixed(0)).toLocaleString('en'));
-            $("#difficulty").text(parseFloat(json.data[0].difficulty).toFixed(2));
+            $("#difficulty").text(json.data[0].difficulty);
```

Comment:
Why are we changing this? If you want this formatted differently then we should add config options for it in a separate PR

Reply:
Fixed in https://github.com/iquidus/explorer/commit/b7180dc9224adcda0c75845e77aab0bfc8734347